js/webform_civicrm_payment.js

```diff
@@ -71,8 +71,20 @@ cj(function($) {
       '</tr>');
     }
     else {
-      $('td+td', $lineItem).html(CRM.formatMoney(amount));
-      $lineItem.data('amount', amount);
+      var hasVAT = 0;
+      var taxPara = 1;
+      var classes = $lineItem.attr('class').split(' ');
+      for (var i = 0; i < classes.length; i++) {
+    	var match = /^vat-/.exec(classes[i]);
+      if(match !== null){
+    		hasVAT = classes[i].substr(classes[i].indexOf("-") + 1);
+    	}
```

Comment:
This seems overly complicated. Any reason you must use css classes and regex instead of just attaching the data directly? Similar to `data-amount` you could add `data-vat` above.